sbin/build.sh

```diff
@@ -831,6 +831,85 @@ fixJavaHomeUnderDocker() {
   fi
 }
 
+addInfoToReleaseFile(){
+  # Extra information is added to the release file here
+  cd $PRODUCT_HOME
+  JAVA_LOC="$PRODUCT_HOME/bin/java"
+  addImplementor
+  addBuildSHA
+  addFullVersion
+  addBuildOS
+  addJVMVariant
+  addJVMVersion
+  # OpenJ9 specific options
+  if [ "${BUILD_CONFIG[BUILD_VARIANT]}" == "${BUILD_VARIANT_OPENJ9}" ]; then
+    addHeapSize
+    addJ9Tag
+  fi
+}
+
+addHeapSize(){ # Adds an identifier for heap size on OpenJ9 builds
+  local heapSize=""
+  if [[ $($JAVA_LOC -version 2>&1 | grep 'Compressed References') ]]; then
+    heapSize="Standard"
+  else
+    heapSize="Large"
+  fi
+  echo -e HEAP_SIZE=\"$heapSize\" >> release
+}
+
+addImplementor(){
+  if [ "${BUILD_CONFIG[OPENJDK_CORE_VERSION]}" == "${JDK8_CORE_VERSION}" ]; then
+    echo -e IMPLEMENTOR=\"AdoptOpenJDK\" >> release
```

Comment:
Can we parameterise this? Other vendors use our build scripts so this might break them, perhaps have it default to Adopt?

Reply:
My intention was to mirror https://github.com/AdoptOpenJDK/openjdk-build/pull/1461/commits/25a8126cb8f6b9923097f9009dbe8e2ed2f13cbb that only affects jdk11+, would a better option be to add a `.make-jdk-any-platform.sh` argument that specifies VENDOR_NAME and defaults to "AdoptOpenJDK"?

Reply:
Or I could follow what was done there and have it only apply when Adopt patches are used.

Reply:
@austin0 just to add, from my point of view, if there will be `IMPLEMENTOR="AdoptOpenJDK"` everywhere - from 8 to 16+, this will make tasks much easier...

Reply:
Added parameter for IMPLEMENTOR.